notebooks/depth_time_series.ipynb

```diff
@@ -0,0 +1,2310 @@
+{
```

Comment:
<p>I find it odd that this plot starts at year 0. Perhaps this is including values with year=NA? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/iobis/pyobis/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I don't know why is the graph starting from zero. But:</p><ul><li>if it would have been the case of nan then those values are not plotted in a seaborn plot</li><li>we have already dropped NaN values when we were defining timeDF in the previous cell</li></ul>